dnf-behave-tests/features/microdnf/install8.feature

```diff
@@ -0,0 +1,19 @@
+@no_installroot
+Feature: microdnf install command on packages
+
+
+Background:
+  Given I delete file "/etc/dnf/dnf.conf"
+    And I delete file "/etc/yum.repos.d/*.repo" with globs
+
+
+@bz1771012
+Scenario: Install package with --nodocs option from local repodata with local packages
+Given I use repository "dnf-ci-fedora"
+  And I execute microdnf with args "--nodocs install man-pages"
+ Then the exit code is 0
+  And microdnf transaction is
+      | Action        | Package                                   |
+      | install       | man-pages-0:4.16-3.fc29.x86_64            |
+  And file "/usr/share/doc/man-pages/README" does not exist
```

Comment:
I think it would be a good idea to install the package also without the flag and check the file is there, because there's a solid chance the file won't be there by a mistake in the scenario, like a typo...